### PR TITLE
1 1 init repo

### PR DESCRIPTION
**Pull request by ngilain**

### CODE REVIEW

**Comment by alavieille** [comment]
:+1:

File: .travis.php.ini [hunk 1]
```diff
@@ -0,0 +1 @@
+extension = "apc.so"
```
Comment (ggrasset) [inline .travis.php.ini:1]: This extension wasn't removed ?

Comment (ngilain) [inline .travis.php.ini:1]: no, it is still used because the build is made on php 5.4 :)

**Comment by ggrasset** [comment]
:+1:

**Pull request merged**